core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -214,10 +360,10 @@ class CommunicationPlan
       will be efficiently migrated.
     */
     template<class ViewType>
-    void createFromExportsAndTopology(
-        const ViewType& element_export_ranks,
-        const std::vector<int>& neighbor_ranks,
-        const int mpi_tag = 1221 )
+    Kokkos::View<int*,memory_space>
```

Comment:
Good catch! The `memory_space` should instead be device type and `int` should be `size_t`. Note that although this function is `public` it is really `private` and an implementation detail. Because we are using class data in a CUDA kernel here we must declare these functions private. This is a known issue we have been discussing with NVIDIA.